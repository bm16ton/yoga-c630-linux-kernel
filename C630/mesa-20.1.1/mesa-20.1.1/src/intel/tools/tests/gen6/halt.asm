(-f0.1.any4h) halt(8) JIP: 96   UIP: 98                         { align1 1Q };
halt(8)         JIP: 2          UIP: 2                          { align1 1Q };
(-f0.1.any4h) halt(16) JIP: 100 UIP: 102                        { align1 1H };
halt(16)        JIP: 2          UIP: 2                          { align1 1H };
