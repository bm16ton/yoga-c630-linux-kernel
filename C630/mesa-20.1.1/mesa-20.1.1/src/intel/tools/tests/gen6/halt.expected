2a 00 76 00 84 1c 00 20 00 00 8d 02 60 00 62 00
2a 00 60 00 84 1c 00 20 00 00 8d 00 02 00 02 00
2a 00 96 00 84 1c 00 20 00 00 8d 02 64 00 66 00
2a 00 80 00 84 1c 00 20 00 00 8d 00 02 00 02 00
